### ISSUE TITLE
Maintainer still needed?

### ISSUE THREAD

**Issue by mattpolito** [bcsgsvn/finance#42]
Is this repo still in need of a maintainer? I would be glad to try and help out.

**Comment by weshatheleopard** [comment]
@mattpolito  go ahead, my [PR](https://github.com/marksweston/finance/pull/39) that fixes a critical bug is still unmerged after 3 years.

**Comment by jarkko** [comment]
@weshatheleopard Yep, an upgrade to Ruby 2.6.0 started causing these with code using `xirr` that used to work with 2.5.3:

```
     ArgumentError:
       invalid value for BigDecimal(): "27875.713148205723+10962.501565059125i"

     ArgumentError:
       invalid value for BigDecimal(): "5619.297046597235-1120.7528551466835i"
```

**Comment by weshatheleopard** [comment]
@jarkko I'm afraid it was an accident and Ruby 2.6 has nothing to do with it -- I had this issue a couple years ago with 2.3(?) as you can see.
